src/test/java/com/odeyalo/sonata/connect/repository/InMemoryPlayerStateRepositoryTest.java

```diff
@@ -28,13 +31,23 @@ static class InMemoryPlayerStateTestEntityGenerator implements TestEntityGenerat
         public PlayerState generateValidEntity() {
             return InMemoryPlayerState.builder()
                     .id(1L)
+                    .repeatState(RepeatState.OFF)
+                    .shuffleState(false)
+                    .playing(true)
+                    .playingType(PlayingType.TRACK)
+                    .user(InMemoryUserEntity.builder().id("mikuuu").build())
                     .build();
         }
 
         @Override
         public PlayerState generateInvalidEntity() {
             return InMemoryPlayerState.builder()
```

Comment:
I should create PlayerStateFaker to re-use entity generation for different impls.